docker/images/interactive/dotnet-interactive/Dockerfile

```diff
@@ -2,31 +2,24 @@ FROM jupyter/base-notebook:ubuntu-18.04
 
 ARG DOTNET_CORE_VERSION=3.1
 
-ENV DOTNET_CORE_VERSION=$DOTNET_CORE_VERSION
-ENV PATH="${PATH}:${HOME}/.dotnet/tools"
-
-ENV DOTNET_RUNNING_IN_CONTAINER=true \
+ENV DOTNET_CORE_VERSION=$DOTNET_CORE_VERSION \
```

Comment:
Per the [Dockerfile Best Practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments),  sort multi-line instructions to improve readability where possible (e.g. cross dependencies)